features/e2e_tests/README.md

```diff
@@ -0,0 +1,39 @@
+# E2E Tests
+
+## Test execution
+
+The way to execute the tests It's quite easy:
+
+- Input:
+```
+bash performance_test.sh
+```
+
+- Output:
+```
+===> Performance Tuning results: (16/8/24) (OK/NOK/TESTS)
+PERF_FEAT_GATE: Test Failed!
+PERF_KUBE_CONFIG_WORKER_RT: Test Failed!
+KERNEL_PREEMPTION: Test Failed!
+KERNEL_REAL_TIME: Test Failed!
+KERNEL_VERSION: Test Failed!
+PERF_KUBE_CONFIG_WORKER_RT: Test Failed!
+PERF_KERN_ARGS_HUGE_PAGES: Test Failed!
+PERF_KERN_ARGS_DEF_HUGE_PAGES: Test Failed!
+```
+
+The result will be something like this `(16/8/24)` where the first field is the successful tests, the second one failed tests, and the third one how many tests has been executed.
```

Comment:
I'd say to better be more explicit, as:
```
Total tests: Z (OK: X, KO: Y)
```

Reply:
If someone more agrees on this I will change it, because I see more readable `NOK` vs `KO`

Reply:
just as an example, this is the output of `go test -v`
```
Ran 13 of 13 Specs in 0.126 seconds
SUCCESS! -- 13 Passed | 0 Failed | 0 Pending | 0 Skipped
```
So I agree to be more explicit and propose `Passed` and `Failed`